app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,247 +11,172 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
-$sidebar-small-width: 75%;
-$sidebar-medium-width: 65%;
-$sidebar-large-width: 45%;
-$sidebar-footer-height: 5em;
-
-.darkswarm {
-  .shop-filters-sidebar {
-    display: flex;
-    flex-direction: column;
-    height: 100%;
-
-    .background {
-      position: fixed;
-      top: 0;
-      right: 0;
-      z-index: 200;
-      height: 100%;
-      width: 100%;
-      background-color: $shop-sidebar-overlay;
-      opacity: 0;
-      transition: opacity $transition-sidebar;
-    }
+.shop-filters-sidebar {
+  .property-selectors {
+    margin-bottom: $sidebar-footer-height + 2em;
+  }
+}
 
-    &.shown {
-      .background {
-        opacity: 1;
-      }
+products {
+  display: block;
 
-      .sidebar, .sidebar-footer {
-        margin-right: 0;
-      }
+  @include breakpoint(tablet) {
+    input.button.right {
+      float: left;
     }
+  }
 
-    .sidebar {
-      position: fixed;
-      top: 0;
-      right: 0;
-      z-index: 210;
-      height: 100%;
-      width: $sidebar-large-width;
-      margin-right: -$sidebar-large-width;
-      background-color: rgba($white, 0.95);
-      padding: 1em;
-      transition: margin $transition-sidebar;
-      overflow-y: scroll;
-
-      .property-selectors {
-        margin-bottom: $sidebar-footer-height + 2em;
-      }
+  @include breakpoint(mobile) {
+    .add_to_cart {
+      margin-top: 2rem;
     }
+  }
 
-    .sidebar-footer {
-      background-color: $grey-800;
-      width: $sidebar-large-width;
-      margin-right: -$sidebar-large-width;
-      height: $sidebar-footer-height;
-      position: fixed;
-      bottom: 0;
-      right: 0;
-      transition: margin $transition-sidebar;
-      padding: 1em;
-
-      button {
-        width: 48%;
+  form {
+    input.small.button.primary.right.add_to_cart {
```

Comment:
this high specificity selecting the element will bring trouble in the future. Why not simply adding a new class to the specific one we want if we fear unwanted elements would get selected?